explainers/subresource-loading.md

```diff
@@ -249,8 +249,8 @@ In the following, `a.js` will be loaded, but `b.js` will be blocked:
 ```
 <script type="webbundle">
 {
-  source: "https://example.com/subresources.wbn",
-  resources: ["https://example.com/script/a.js",
+  "source": "https://example.com/subresources.wbn",
+  "resources": ["https://example.com/script/a.js",
               "https://example.com/b.js"]
```

Comment:
nit: Add 2 more spaces (to maintain indent)

Reply:
Done.